src/SessionSetup/ConnectToLocalWidget.ui

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ConnectToLocalWidget</class>
+ <widget class="QWidget" name="ConnectToLocalWidget">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>748</width>
+    <height>51</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <property name="accessibleName">
+   <string>Orbit Profiler</string>
```

Comment:
Nit: Shouldn't this be somehow connected to the connected to the "connect to local"?

Reply:
I created that widget by modifying a copy of `ConnectToStadiaWidget` and this is a leftover from that. I removed it here (and the weird widget title `Form`). I guess its somehow used for e2e tests. The `ConnectToStadiaWidget` will also go away in its current form, so this will be resolved shortly as well.